app/assets/javascripts/modules/calendars/allocations.es6

```diff
@@ -38,19 +41,11 @@
       let title;
 
       if (title = prompt(`Name of holiday period for ${resource.title}?`)) {
-        console.log('Holiday title', title);
-        console.log('Holiday start', start.format());
-        console.log('Holiday end', end.format());
-        console.log('Holiday guider id', resource.id);
-        console.log('Holiday guider name', resource.title);
-
-        // @todo Jquery ajax request to POST a new holiday
-        // in the onsuccess method we will need to call
-        // this.$el.fullCalendar('refetchEvents');
-        // this.showAlert('.alert-success');
-        //
-        // or call this.showAlert('.alert-danger');
-        // on failure
+        this.$holidayForm.find('.js-user-id').val(resource.id);
```

Comment:
Mostly cause I started to get all paranoid about CSRF and thought I should do the safest thing.
I'm not 100% sure that's valid I should probably learn rails a bit more 🤑